python/py-prompt_toolkit/Portfile

```diff
@@ -4,22 +4,22 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-prompt_toolkit
-version             1.0.14
+version             1.0.15
 license             Permissive
 platforms           darwin
 supported_archs     noarch
 maintainers         {@xeron gmail.com:xeron.oskom} openmaintainer
 description         Library for building powerful interactive command lines in Python
 long_description    ${description}
 
-python.versions     27 34 35 36
+python.versions     27 34 35 36 37
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  c0c4aa0c917981767d45abf76fdff00c0fb9aa8d \
-                    sha256  cc66413b1b4b17021675d9f2d15d57e640b06ddfd99bb724c73484126d22622f
+checksums           rmd160  195e01e80e62e631f7c4b12d4f2cd5d4481a0752 \
+                    sha256  858588f1983ca497f1cf4ffde01d978a3ea02b01c8a26a8bbc5cd2e66d816917
```

Comment:
It is now customary to put in a "size" field in addition to the rmd160 and sha256.